src/sentry/auth/system.py

```diff
@@ -14,17 +18,17 @@
 ]
 
 
-def is_internal_ip(request):
+def is_internal_ip(request: HttpRequest) -> bool:
     ip = ipaddress.ip_address(str(request.META["REMOTE_ADDR"]))
     return any(ip in net for net in INTERNAL_NETWORKS)
 
 
-def get_system_token():
+def get_system_token() -> str:
     token = options.get("sentry:system-token")
     if not token:
         token = uuid4().hex
         options.set("sentry:system-token", token)
-    return token
+    return token  # type: ignore[no-any-return]
```

Comment:
Why does this need a `# type: ignore`?

Reply:
the type checker does not understand django options so (as the type ignore comment says) it does not allow `Any` to be returned when a type is specified